bp_be/src/include/bp_be_internal_if_defines.vh

```diff
@@ -94,6 +94,8 @@
     logic                                    ex1_v;                                                \
     logic [rv64_eaddr_width_gp-1:0]          ex1_pc;                                               \
                                                                                                    \
+    logic                                    fe_exception_v;                                       \
```

Comment:
Don't need this either.  The line in detector should be 
    // Detect misprediction
    mispredict_v = (calc_status.ex1_pc != expected_npc_i);